Components/HeroSlider/script.js

```diff
@@ -0,0 +1,65 @@
+import $ from 'jquery'
+import Swiper from 'swiper'
+import 'swiper/css/swiper.min.css'
+
+class HeroSlider extends window.HTMLDivElement {
+  constructor (...args) {
+    const self = super(...args)
+    self.init()
+    return self
+  }
+
+  init () {
+    this.$ = $(this)
+    this.props = this.getInitialProps()
+    this.resolveElements()
+  }
+
+  getInitialProps () {
+    let data = {}
+    try {
+      data = JSON.parse($('script[type="application/json"]', this).text())
+    } catch (e) {}
+    return data
+  }
+
+  resolveElements () {
+    this.$slider = $('[data-slider]', this)
+    this.$sliderItems = $('.swiper-slide', this)
+    this.$buttonNext = $('[data-slider-button="next"]', this)
+    this.$buttonPrev = $('[data-slider-button="prev"]', this)
+    this.$pagination = $('[data-slider-pagination]', this)
+  }
+
+  connectedCallback () {
+    if (this.$sliderItems.length > 1) {
+      this.initSlider()
+    }
+  }
+
+  initSlider () {
+    const { options } = this.props
+    const config = {
+      a11y: options.a11y,
```

Comment:
Is this property actually set?

Reply:
Yes, it's pulled through from Translatable Options -> SliderOptions